tests/post_deploy_functional/main_test.go

```diff
@@ -31,11 +31,6 @@ func TestFirewallModule(t *testing.T) {
 		SetTestConfig(&testimpl.ThisTFModuleConfig{}).
 		SetTestConfigFolderName(testConfigsExamplesFolderDefault).
 		SetTestConfigFileName(infraTFVarFileNameDefault).
-		SetTestSpecificFlags(map[string]types.TestFlags{
-			"complete": {
```

Comment:
complete should be replaced with the actual example name "firewall_policy" in this case